optuna/storages/_redis.py

```diff
@@ -140,7 +141,7 @@ def create_new_study(self, study_name: Optional[str] = None) -> int:
                 "study_id:{:010d}:directions".format(study_id),
                 pickle.dumps([StudyDirection.NOT_SET]),
             )
-
+            # TODO(wattlebirdaz): Replace StudySummary with FrozenStudy
```

Comment:
A nit.
```suggestion
            # TODO(wattlebirdaz): Replace StudySummary with FrozenStudy.
```

I agree with you. If we simply replace `StudySummary` with `FrozenStudy`, existing users cannot read the studies from RedisStorage. So, we may need to provide migration code, which will make the PR complicated.